Oracle/PCORI_MEDS_SCHEMA_CHANGE_ora.sql

```diff
@@ -13,38 +53,41 @@ select * from
 with
 cui(c_fullname,pcori_cui,c_hlevel) as
 (
-    select c_fullname,pcori_cui,c_hlevel from pcornet_med where pcori_cui is not null
+    select c_fullname,pcori_cui,c_hlevel from I2B2METADATA.pcornet_med where pcori_cui is not null
    union all
     select m.c_fullname, cui.pcori_cui, m.c_hlevel
-    from pcornet_med m inner join cui on cui.c_fullname=m.c_path where m.pcori_cui is null
+    from I2B2METADATA.pcornet_med m inner join cui on cui.c_fullname=m.c_path where m.pcori_cui is null
```

Comment:
Is adding the `I2B2METADATA.` schema prefix really a fix? In what way was it broken? The schema we use has a different name. We (@kumc-bmi) went with a parameter; e.g. `"&&i2b2_data_schema".OBSERVATION_FACT`

https://github.com/kumc-bmi/i2p-transform/blob/master/Oracle/PCORNetLoader_ora.sql#L66